pages/words/[id].js

```diff
@@ -22,24 +27,65 @@ export default function SingleEntry() {
             word: editedEntry.word,
             language: editedEntry.language,
             translated: editedEntry.translated,
+            notes: editedEntry.notes,
           };
         }
         return translation;
       })
     );
+    setIsShowMode(true);
   }
 
   return (
     <>
       <main>
-        <h1>Edit word</h1>
-        <CancelDeleteButton onClick={() => router.back()} />
-        {!entry ? null : (
-          <Form
-            entry={entry}
-            isEditMode={true}
-            onSubmitEvent={handleEditEntry}
-          />
+        {isShowMode ? (
+          <>
+            <h1>Word entry </h1>
+            <GoBackButton onClick={() => router.push("/words")} />
+            <EditButton onClick={() => setIsShowMode(false)}></EditButton>
```

Comment:
This can be self-closing:

```suggestion
            <EditButton onClick={() => setIsShowMode(false)} />
```

Reply:
Done